### PR TITLE
unload avatar, debug account hashes, improve rigManager

### PR DESCRIPTION
**Pull request by ChrisEddy**

### CODE REVIEW

File: edit.js [hunk 1]
```diff
@@ -121,7 +121,11 @@ orbitControls.target.copy(camera.position).add(new THREE.Vector3(0, camera.posit
 orbitControls.update();
 
 loginManager.addEventListener('avatarchange', async (e) => {
-  rigManager.setLocalAvatarUrl(`${storageHost}/${e.data}`);
+  if(e.data) {
+    rigManager.setLocalAvatarUrl(`${storageHost}/${e.data}`);
+  } else {
+    rigManager.addLocalRig();
```
Comment (avaer) [inline edit.js:127]: Probably should be explicit `null`.

File: edit.js [hunk 2]
```diff
@@ -121,7 +121,11 @@ orbitControls.target.copy(camera.position).add(new THREE.Vector3(0, camera.posit
 orbitControls.update();
 
 loginManager.addEventListener('avatarchange', async (e) => {
-  rigManager.setLocalAvatarUrl(`${storageHost}/${e.data}`);
+  if(e.data) {
```
Comment (avaer) [inline edit.js:124]: Linting.

**Review by avaer — COMMENTED** [review]
Just some suggestions.

**Comment by ChrisEddy** [comment]
fixed

**Pull request merged**